compiler/vmdef.nim

```diff
@@ -261,11 +261,23 @@ type
     config*: ConfigRef
     graph*: ModuleGraph
     oldErrorCount*: int
+    profile*: Profile
+
+  FileLine* = object
```

Comment:
move all profiling specific code to a separate module `compiler/profileutils`

Reply:
It's `vmprofiler.nim` for now.